packages/Sortable/src/components/SortableItem/SortableItem.js

```diff
@@ -50,12 +53,9 @@ const SortableItem = ({ children, index, hasNumbers, onRemove }) => {
           </div>
           {onRemove && (
             <div css={itemCloseStyles} data-pka-anchor="sortable.item.remove">
-              <Button.Close
-                a11yText={I18n.t("sortable.aria_remove")}
```

Comment:
Is it save to remove `sortable.aria_remove` string? And is there an announcement on the icon now?

Reply:
Good catch